scp/lib/time_table.dart

```diff
@@ -339,11 +355,39 @@ class TimeTableState extends State<TimeTable> {
                     ),
                     Align(
                       alignment: Alignment.bottomCenter,
-                      child: ElevatedButton(
-                          child: Text('Location'),
-                          onPressed: () {
-                            launchMap(periodDetail.location);
-                          }),
+                      child: SizedBox(
+                        width: screenWidth*0.4,
+                        child: RaisedButton(
```

Comment:
Are we planning to move to elevated button anytime soon? Ig this is cause for the build to fail @asutoshranjan !!